### DISCUSSION TITLE
New notification unable to be cleared, and positioning may be off

### DISCUSSION THREAD

**Discussion by ghost** [community/community#56813] [Mobile] [answered]
See screenshot of notification dot on issue in the issue tracker.

The blue dot is also not central to the open issue green dot, not sure if this is intentional.

![image](https://github.com/community/community/assets/95468948/a7d945f4-9579-4bb1-9b4f-f74b07f62e38)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.114.0 (99800193)
Device: iPhone 13 Pro (iOS 16.6)
</details>

**Comment by Iron-Ham** [comment, marked as answer]
Hi @jslawler, thank you for bringing this to our attention. I'll open up an internal ticket and start working on it right away.
